pages/calculator/[slug].js

```diff
@@ -147,26 +163,53 @@ export default function CalculatorSlugRoute({ page, calculatorConfig }) {
                 >
                   {calculatorConfig.feedback.linkText}
                 </Button>
-                {/* check another conviction */}
-                <Link
-                  sx={{ textAlign: 'center' }}
-                  href={
+
+                <Box sx={{
+                  display: 'flex', flexDirection: 'column', gap: 1.5, marginLeft: isMobile ? 0 : 7,
+                }}
+                >
+                  {/* check another conviction */}
+                  <Link
+                    sx={{ textAlign: 'center', whiteSpace: 'nowrap' }}
+                    href={
                     calculatorConfig.checkAnotherConviction.linkTo.slug.current
                   }
-                >
-                  <Box
-                    sx={{
-                      display: 'flex',
-                      alignItems: 'center',
-                      justifyContent: 'center',
-                      gap: 1,
+                  >
+                    <Box
+                      sx={{
+                        display: 'flex',
+                        gap: 1,
+                      }}
+                    >
+                      <HistoryIcon />
+                      {calculatorConfig.checkAnotherConviction.linkText}
+                    </Box>
+                  </Link>
+
+                  <Link
+                    href={
+                  calculatorConfig.checkAnotherConviction.linkTo.slug.current
```

Comment:
The variable being used here, while being the correct link, is really confusing because we already have the check another conviction button declared above.